games/m.yaml

```diff
@@ -183,9 +185,34 @@
     lang: C++
     updated: 2016-04-20
 
+- name: 'Magic: The Gathering Online'
+  meta:
+    genre: [Strategy]
```

Comment:
TBS is better suited here. Maybe we should also consider a "Card game" main genre which is more exact as a follow-up issue after this is merged.